### PR TITLE
Use int64_t in EnvInfo for block gas limit

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

**Comment by winsvega** [comment]
merged into https://github.com/ethereum/cpp-ethereum/pull/3065

**Comment by chfast** [comment]
WTF? Have you seen it contains other changes?

**Comment by winsvega** [comment]
Oops havent saw your second commit, sorry

File: libevm/ExtVMFace.h [hunk 1]
```diff
@@ -173,7 +173,10 @@ class EnvInfo
 		m_author(_current.author()),
 		m_timestamp(_current.timestamp()),
 		m_difficulty(_current.difficulty()),
-		m_gasLimit(_current.gasLimit()),
+		// Trim gas limit to int64. convert_to used explicitly instread of
```
Comment (bobsummerwill) [inline libevm/ExtVMFace.h]: instread typo

File: libevm/ExtVMFace.h [hunk 2]
```diff
@@ -183,7 +186,10 @@ class EnvInfo
...
```
Comment (bobsummerwill) [inline libevm/ExtVMFace.h]: instread

File: libevm/ExtVMFace.h (hunk 1 above)
Comment (chfast) [inline libevm/ExtVMFace.h]: I make this mistake every single time.

**Comment by bobsummerwill** [comment]
LTGM in general.   Please just fix the typo in the comment and merge-at-will.

The automation breaks do not appear to be related, but are just automation breaks which *I* need to keep chasing.

Please ignore "run out of time ones".   Thanks!

**Pull request merged**